build-support/bin/cherry_pick/BUILD

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package_json(
+    name="package.json",
+    scripts=[node_test_script()]
```

Comment:
I think the `node_test_script` is unnecessary boilerplate in this case. The backend assumes you'd want the `"test"` script by default as what pants runs.

`node_test_script` is for when you want some other script to run, or extra env configuration and whatnot